internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/service/VeracodeScanner.java

```diff
@@ -64,31 +66,36 @@
     public VeracodeScanner() throws IOException {
         loadConfiguration();
         VeracodeCommand.Options options;
-        scanContext = new ScanContext();
+        //        scanContext = getScanContext();
```

Comment:
Is this intentional?

Reply:
Yah. We can remove this. With resilience capability, new scancontext object will be created while validating.